indico/modules/events/abstracts/blueprint.py

```diff
@@ -23,8 +23,9 @@
                                                                          RHEditEmailTemplateText, RHEmailTemplateList,
                                                                          RHDeleteEmailTemplate, RHSortEmailTemplates,
                                                                          RHEmailTemplateREST)
-from indico.modules.events.abstracts.controllers.management import (RHAbstracts, RHAbstractList,
-                                                                    RHAbstractListCustomize, RHAbstractListStaticURL,
+from indico.modules.events.abstracts.controllers.management import (RHAbstractList,
+                                                                    RHAbstracts, RHAbstractListCustomize,
+                                                                    RHAbstractListStaticURL,
```

Comment:
And make rebase conflicts more likely. :/

I'd rather change the imports to just import modules here later once this and the other PRs are merged and then use e.g. `management.RHWhatever` in the rule definitions.